frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -595,6 +608,17 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
         />
       </div>
       {suggestedNamespaceDetails}
+      {suggestedNamespaceTemplateName &&
```

Comment:
Shouldn't it be `operatorSuggestedNamespace` since thats contains the namespace from the template annotation(if set), otherwise use the namespace name from the annotation(if set)
```suggestion
      {operatorSuggestedNamespace &&
```

here and below...

Reply:
Updated with change